fe/fe-core/src/main/java/org/apache/doris/nereids/types/TimeV2Type.java

```diff
@@ -27,7 +29,9 @@
  */
 public class TimeV2Type extends PrimitiveType implements RangeScalable {
```

Comment:
maybe we should remove TimeV2Type and retain TimeType

Reply:
I think TimeV2Type is more proper. when we use different branch or timepoint of codes, if changing to TimeType, maybe we will confuse between old V1 type and new V2 type. keeping version growth also acts like what we did for Decimal types